server.js

```diff
@@ -15,6 +15,7 @@ app.get('/', (req, res) => res.json({ msg: 'Welcome to the CIP API' }));
 
 // Define routes
 app.use('/courses', require('./routes/courses'));
+app.use('/courseColumnOptions', require('./routes/courseColumnOptions'));
```

Comment:
Let's keep file names to be `kebab-case` instead `courseColumnOptions.js` -> `course-column-options.js`